### PR TITLE
Adding DataONE to integrations.rst

### PR DESCRIPTION
**Pull request by iannesbitt**
**What this PR does / why we need it**: This addition provides information for Dataverses about how to federate with DataONE.

**Which issue(s) this PR closes**: n/a

Closes #

**Special notes for your reviewer**: Thank you to @pdurbin for the suggestion!

**Suggestions on how to test this**: n/a

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**: n/a

**Is there a release notes update needed for this change?**: No, this is a very small change to the documentation.

**Additional documentation**: n/a

### CODE REVIEW

File: doc/sphinx-guides/source/admin/integrations.rst [hunk 1]
```diff
@@ -214,6 +214,11 @@ Geodisy
 
 `Geodisy <https://researchdata.library.ubc.ca/find/geodisy>`_ will take your Dataverse installation’s data, search for geospatial metadata and files, and copy them to a new system that allows for visual searching. Your original data and search methods are untouched; you have the benefit of both. For more information, please refer to `Geodisy's GitHub Repository. <https://github.com/ubc-library/geodisy>`_
 
+DataONE
++++++++
+
+`DataONE <https://dataone.org/>`_ is a community driven program providing access to data across multiple `member repositories <https://www.dataone.org/network/>`_, supporting enhanced search and discovery of Earth and environmental data. Membership is free and is most easily achieved by providing schema.org data via `science-on-schema.org <https://science-on-schema.org>`_ metadata markup on dataset landing pages, support for which is native in Dataverse. Learn more about joining the network `here <https://www.dataone.org/jointhenetwork/>`_.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/integrations.rst:220]:
```suggestion
`DataONE <https://dataone.org/>`_ is a community driven program providing access to data across multiple `member repositories <https://www.dataone.org/network/>`_, supporting enhanced search and discovery of Earth and environmental data. Membership is free and is most easily achieved by providing schema.org data via `science-on-schema.org <https://science-on-schema.org>`_ metadata markup on dataset landing pages, support for which is native in Dataverse. Dataverse installations are welcome `join the network <https://www.dataone.org/jointhenetwork/>`_ to have their datasets included.
```

This is a great write up as-is but the suggestion above is intended to make it a little more clear to someone running Dataverse that they are welcome to join and why. I'm not sure what the best wording would be! 😅

**Review by pdurbin — COMMENTED** [review]
Overall, looks great. The HTML can be previewed here: https://dataverse-guide--10192.org.readthedocs.build/en/10192/admin/integrations.html#dataone

I did leave a suggestion. Thanks, @iannesbitt!

**Comment by iannesbitt** [comment]
Looks great, thank you @pdurbin! Feel free to merge when you're ready.

**Review by pdurbin — APPROVED** [review]
Looks great. Will merge. Thanks, @iannesbitt!

**Pull request merged**